backstage/plugins/knative-event-mesh-backend/src/providers/knativeEventMeshProcessor.test.ts

```diff
@@ -30,6 +30,8 @@ describe('KnativeEventMeshProcessor', () => {
                         'metadata.namespace':string,
                         'metadata.annotations.backstage.io/kubernetes-id':string,
                     },
+                    cursor: string,
+                    limit: number
```

Comment:
Can you add a new test case?

The new test case should do these:
- `limit=5`
- Return 5 items in the mock API call, also return a cursor
- Test if the processor makes a 2nd call, with the cursor